.travis.yml

```diff
@@ -1,11 +1,9 @@
 language: python
 jdk: openjdk8
 dist: trusty
-sudo: false
 cache:
   pip: true
 python:
-  - "2.7"
```

Comment:
Might make sense to swap this out for:

```yaml
  - "3.5"
  - "3.6"
  - "3.7"
```

Reply:
Good idea.

However, Python 3.7 requires `dist: xenial` on Travis, and a quick test shows more work is needed to test against Xenial, so I'll only add 3.6 as part of this PR and 3.7 can be dealt with in another issue.

https://travis-ci.org/hugovk/datacompy/builds/591518867

Reply:
Sounds good! Thank you Hugo.